cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/ShopPartialBidCalculationInput.view.yaml

```diff
@@ -43,28 +43,15 @@ properties:
   shopResults:
     type:
       space: '{{powerops_type_space}}'
-      externalId: SHOPResult
+      externalId: SHOPResultPriceProd
     source:
       space: '{{powerops_models}}'
-      externalId: SHOPResult
+      externalId: SHOPResultPriceProd
       version: '{{version}}'
       type: view
     direction: outwards
     name: shopResults
-    description: An array of shop results.
-    connectionType: multi_edge_connection
-  alerts:
-    type:
-      space: '{{powerops_type_space}}'
-      externalId: calculationIssue
-    source:
-      space: '{{powerops_models}}'
-      externalId: Alert
-      version: '{{version}}'
-      type: view
-    direction: outwards
-    name: alerts
-    description: An array of calculation level Alerts.
+    description: An array of shop results with price/prod timeserires pairs for all plants included in the respective shop scenario
     connectionType: multi_edge_connection
   marketConfiguration:
     container:
```

Comment:
In this view, it would be great to have a boolean property called `step_enabled` that decides over how to handle the volumes, forward-fill or interpolation.